### PR TITLE
Implements Cocos2d's getNumberOfRunningActions() method as number_of_running_actions

### PR DESCRIPTION
**Pull request by prestia**
Tested on iOS builds. I do not have an Android setup.

### CODE REVIEW

File: src/node.cpp [hunk 1]
```diff
@@ -437,6 +437,16 @@ node_unschedule(VALUE rcv, SEL sel, VALUE key)
     return rcv;
 }
 
+/// @method #number_of_running_actions
+/// @return [Integer] the number of running actions for the node.
+
+static VALUE
+
```
Comment (lrz) [inline src/node.cpp]: It looks like there is an empty line here?

File: src/rubymotion-apple.h [hunk 2]
```diff
@@ -29,6 +29,7 @@ VALUE rb_uint2big(VALUE);
 long rb_num2int(VALUE);
 #define FIX2INT(x) ((int)FIX2LONG(x))
 #define NUM2INT(x) (FIXNUM_P(x) ? FIX2INT(x) : rb_num2int((VALUE)x))
+#define SSIZET2NUM(v) LONG2NUM(v)
```
Comment (lrz) [inline src/rubymotion-apple.h]: Could we rename this `SSIZE2NUM` instead? (basically drop the `T` from the name)

**Comment by lrz** [comment]
Thanks for the contribution @prestia! I added a couple comments :)

File: src/rubymotion-apple.h (hunk 2 above)
Comment (prestia) [inline src/rubymotion-apple.h]: We can. I used `SSIZET2NUM` to remain consistent with the [Ruby C API](https://docs.ruby-lang.org/en/trunk/extension_rdoc.html), but defer to you. Let me know either way!

**Comment by prestia** [comment]
Thanks, @lrz! I fixed one and replied to the other.

File: src/rubymotion-apple.h (hunk 2 above)
Comment (lrz) [inline src/rubymotion-apple.h]: Oh very good point, I wasn't aware of that! Let's definitely use the first version then :)

**Comment by lrz** [comment]
Thanks again for the contribution :-)

**Pull request merged**